src/components/About.js

```diff
@@ -1,6 +1,11 @@
-import React from 'react'
+import React from 'react';
+import UserServiceTest from './UserServiceTest';
 
- function About() {
-  return <h1>About page  </h1>
+function About() {
```

Comment:
I wonder if instead of the About page if it would make things easier (less likely to conflict) if you created a new page on which to include the `UserServiceTest` since it is just an example/demonstration of how things work

Reply:
That's my bad, I got a bit lazy when looking for a place to put the test component.  I'll split it out into a test page before finalizing the merge.